gen/wasm.js

```diff
@@ -256,7 +256,13 @@ async function __wbg_init(input) {
     if (wasm !== undefined) return wasm;
 
     if (typeof input === 'undefined') {
-        input = new URL('wasm_bg.wasm', import.meta.url);
+        try {
+            input = new URL('wasm_bg.wasm', import.meta.url);
+        } catch {
+            // try with dynamic import - CF workers runtime does not have import.meta.url
+            const module = await import('./wasm_bg.wasm')
+            input = module.default
+        }
```

Comment:
Looks like we arrive at this code path form the `load` call here

https://github.com/web3-storage/fr32-sha2-256-trunc254-padded-binary-tree-multihash/blob/95cb375bbb2a5d8610191598a306d9ee97320335/src/lib.js#L5-L6

It also appears to me that we could avoid the problem if we pass `input` instead to the load in which case it will land at this code path

https://github.com/web3-storage/fr32-sha2-256-trunc254-padded-binary-tree-multihash/blob/95cb375bbb2a5d8610191598a306d9ee97320335/gen/wasm.js#L263-L269

That is to say we should deal with the platform variation where we call `load` instead which is not a generated code and will be less error prone.

Reply:
Perhaps we could do something like this instead in `lib.js`

```js
import bytecode from "./wasm_bg.js"

load(bytecode.buffer)
```

I believe it will skip all the conditionals and go right into `__wbg_load` where it will skip condition again and just do this

https://github.com/web3-storage/fr32-sha2-256-trunc254-padded-binary-tree-multihash/blob/95cb375bbb2a5d8610191598a306d9ee97320335/gen/wasm.js#L193-L194

Reply:
```
import bytecode from "./wasm_bg.js"

load(bytecode.buffer)
```